### ISSUE TITLE
Consider adding departure time

### ISSUE THREAD

**Issue by dabreegster** [dabreegster/odjitter#21]
If the CSV input has something like a `departure_seconds` column, the jittered output could have this too. For each output row, the departure time would be jittered somehow -- maybe a uniform or normal distribution centered around the input time? We would need extra config and flags (with default values) to specify all of this.

@lucasccdias, is my understanding correct? How specifically would you want to jitter `departure_seconds`?

I'm hesitant to add this feature, because I'm not convinced that it will be easy for the user to learn and specify a bunch of extra command-line flags to say how they want to transform time. Instead, why couldn't they add departure time on their end? In other words:

1) Write the desire line CSV file
2) Call odjitter on it
3) Read the output GeoJSON file and add a departure time property, using whatever logic they want

One question is how departure time is determined. Does any of the input desire line data have something like this? How is it specified -- maybe just the hour range that a bunch of trips go from zone1 to zone2? If so, maybe what we should instead do is make it easy to match up the jittered GeoJSON output with the original input, and have some kind of lookup key, or just copy over the departure time property, and do the jittering on that elsewhere.

And backing up a little more, I think the motivation for this feature request was to generate A/B Street [scenarios](https://a-b-street.github.io/docs/tech/dev/formats/scenarios.html), either with `abstr` or not. If so, it could be helpful to understand how we want to do that. Part of odjitter input is weighted subpoints, and there's more in-progress code within A/B Street to generate these weights for the exact purpose of creating scenarios. If our ultimate aim is to create scenarios from raw desire line data, we have a spectrum of options how to do it -- some of them using odjitter, some of them directly calling this other pipeline.

**Comment by Robinlovelace** [comment]
Yes, it would be great to create 'scenario.json' files, worth splitting out as a separate issue at some point.

**Comment by Robinlovelace** [comment]
One for Tuesday afternoon?
